packages/preview/chuli-cv/0.1.0/template/cv.typ

```diff
@@ -0,0 +1,174 @@
+#import "@preview/chuli-cv:0.1.0": *
+#import "@preview/fontawesome:0.1.0": *
+
+#show: cv
+
+#let icons = (
+  phone: fa-phone(),
+  homepage: fa-home(fill: colors.accent),
+  linkedin: fa-linkedin(fill: colors.accent),
+  github: fa-github(fill: colors.accent),
+  xing: fa-xing(),
+  mail: fa-envelope(fill: colors.accent),
+  book: fa-book(fill: colors.accent),
+  cook: fa-utensils(fill: colors.accent),
+  bike: fa-biking(fill: colors.accent),
+  game: fa-gamepad(fill: colors.accent),
+  robot: fa-robot(fill: colors.accent),
+  bed: fa-bed(fill: colors.accent),
+  write: fa-pen-to-square(fill: colors.accent),
+  talk: fa-comments(fill: colors.accent),
+  code: fa-code(fill: colors.accent),
+  paint: fa-paintbrush(fill: colors.accent),
+  music: fa-music(fill: colors.accent),
+  friends: fa-users(fill: colors.accent),
+  beer: fa-beer(fill: colors.accent),
+)
+
+#header(
+  full-name: [Faye Valentine],
+  job-title: [Bounty Hunter],
+  socials: (
+    (
+      icon: icons.github,
+      text: [username],
+      link: "https://github.com/xxx"
+    ),
+    (
+      icon: icons.mail,
+      text: [username\@gmail.com],
+      link: "mailto://username@gmail.com"
+    ),
+    (
+      icon: icons.linkedin,
+      text: [Faye Valentine],
+      link: "https://linkedin.com/in/npujolm/"
+    ),
+    (
+      icon: icons.homepage,
+      text: [Earth],
+      link: "#"
+    ),
+  ),
+  profile-picture: image("media/avatar.png")
```

Comment:
The `media` directory won't be copied to projects using this template, so this wouldn't work. If you want to include an example image, it should go in the `template` directory (it can be in a dedicated sub-directory if you want).

Reply:
I think I might have missed something. `media` folder is already located inside `template` folder, isn't that how it should be?
![image](https://github.com/typst/packages/assets/18596342/1395654e-583a-4908-bc7b-bdd0445f778f)

Reply:
Yes, my bad, you are right.